packages/schema/lib/schemas/AppSchema.js

```diff
@@ -106,6 +106,21 @@ module.exports = makeSchema(
         description: 'Top-level app options',
         $ref: AppFlagsSchema.id,
       },
+      // inputFields: {
```

Comment:
@kola-er I was not sure what where the split sub-schemas should be used at the top level, this worked to generated the docs and exported-schema but didn't feel right 🤔

Reply:
They should only show up where they are referenced/used. Currently, we don't use FieldSchema at the app level. It is easier to locate those places by searching for `inputFields` and `outputFields`. You would reference these new schemas there instead of the `/DynamicFieldsSchema` currently being used. And reference the new AuthfieldsSchema in `AuthenticationSchema` for the fields property.